src/API/AbstractRepoAPI.php

```diff
@@ -6,5 +6,13 @@
 
 abstract class AbstractRepoAPI extends AbstractAPI
 {
-    protected $api_prifix = '/api/repo/v1';
-}
\ No newline at end of file
+    const API_PRIFIX_BASE = '/api/repo/v1';
+    protected $api_prifix = self::API_PRIFIX_BASE;
+
+    /**
+     * @param $namespace
+     */
+    function setOrganisationNamespace($namespace){
```

Comment:
Given this function is the same for all of these classes, is it not worth adding it to the `AbstractAPI` class instead so these inherit it?

Also, this is missing a `public` function declaration.

Reply:
I thought of that but it needs to know the 'api_prifix' to append the namespace otherwise if it was called a second time it would break. And i didn't think it needed to be public since the NodeCluster / Stack API classes extend it